lib/twilio/uuid.ts

```diff
@@ -0,0 +1,39 @@
+import * as md5 from 'md5';
+import { NotSupportedError } from '../twilio/errors';
+
+export function generateUuid(): string {
```

Comment:
Why are we exporting it? For unit tests? If so, we should still keep it private. We can hit this function by testing generateVoiceEventSid instead.

Reply:
Hmm, I can see us using in other things down the line maybe, but we can export it then. I'll make it non-exported.

Reply:
Right. We should not export anything that is not needed. Only do it when you need it. [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) 😉